storedvalue/contract.go

```diff
@@ -1,22 +1,113 @@
 package storedvalue
 
+import (
+	"encoding/binary"
+	"fmt"
+
+	"github.com/hdac-io/casperlabs-ee-grpc-go-util/protobuf/io/casperlabs/casper/consensus/state"
+)
+
+const (
+	PROTOCOL_VERSION_MAJOR_LENGTH = 4
+	PROTOCOL_VERSION_MINOR_LENGTH = 4
+	PROTOCOL_VERSION_PATCH_LENGTH = 4
+	PROTOCOL_VERSION_LENGTH       = PROTOCOL_VERSION_MAJOR_LENGTH + PROTOCOL_VERSION_MINOR_LENGTH + PROTOCOL_VERSION_PATCH_LENGTH
+)
+
 type Contract struct {
 	Body            []byte          `json:"body"`
-	NamedKeys       []NamedKey      `json:"named_key"`
+	NamedKeys       NamedKeys       `json:"named_key"`
```

Comment:
doesn't it matter misspelling? `json:"named_keys"`

Reply:
https://github.com/hdac-io/casperlabs-ee-grpc-go-util/pull/17/commits/54c0b363fb27b4c8e661d85131f3dfe0856c0055